setup/setup.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable */
```

Comment:
That's not good. Remove this and disable rule per line instead if needed.

Also add the `setup` folder to the eslint and prettier scripts.

Reply:
removed..

Reply:
Added the setup folder. But `prettier` is hell... not sure if this makes code prettier, and interpreting the error msg is not fun ⏎↹↹↹`  ??? Return with three tabs in the middle of my function name? 
I am using VsCode.. will try it manually..

Reply:
Aha, double click on the error --> Fix all. That's easy.